charts/argo/templates/server-deployment.yaml

```diff
@@ -34,33 +33,28 @@ spec:
           {{- if .Values.server.extraArgs }}
           {{- toYaml .Values.server.extraArgs | nindent 10 }}
           {{- end }}
+          {{- if .Values.singleNamespace }}
+          - "--namespaced"
+          {{- end }}
           image: "{{ .Values.images.namespace }}/{{ .Values.images.server }}:{{ default .Values.images.tag .Values.server.image.tag }}"
           imagePullPolicy: {{ .Values.images.pullPolicy }}
           {{- if .Values.server.podPortName }}
           ports:
           - name: {{ .Values.server.podPortName }}
-            ports:
             containerPort: 2746
-            readinessProbe:
```

Comment:
I take all this was broken? I am still unclear of the purpose of `podPortName` to switch this on and off and why not just always expose the named port? but might pay not to change things now.

Reply:
I just fixed the indents here, it was misaligned. Somebody probably had a use case for it as some point so it was added.

Reply:
Yup makes sense!